drivers/storage/portworx/component/monitoring.go

```diff
@@ -1,8 +1,11 @@
 package component
 
 import (
+	"context"
```

Comment:
Let's extract the Grafana components into a different component called grafana.
The main reason being the trigger/flag for grafana is different from the prometheus stack. In the current state, the `IsEnabled` method will allow grafana installation even if just prometheus is enabled.